app/src/main/res/values/strings.xml

```diff
@@ -894,6 +902,55 @@
     <string name="meteoam_weather_text_sand_storm" translatable="false">@string/common_weather_text_sand_storm</string>
     <!-- Example: Heavy rain and thunder -->
     <string name="metno_weather_text_andthunder">%s and thunder</string>
+    <!-- Weather texts for HKO -->
+    <string name="hko_weather_text_sunny" translatable="false">@string/common_weather_text_clear_sky</string>
+    <string name="hko_weather_text_sunny_periods">Sunny periods</string>
+    <string name="hko_weather_text_sunny_intervals">Sunny intervals</string>
+    <string name="hko_weather_text_sunny_periods_with_a_few_showers">Sunny periods with a few showers</string>
+    <string name="hko_weather_text_sunny_intervals_with_showers">Sunny intervals with showers</string>
```

Comment:
Missing `translatable="false"` for a few strings

Reply:
I thought people _may_ want to translate these, but I'll just lock them down as `translatable="false"` as you suggested.